contrib/aks-engine/templates/k8s.json.j2

```diff
@@ -31,7 +31,7 @@
                     ],
                     "config": {
                         "scan-interval": "1m",
-                        "expendable-pods-priority-cutoff": "-1200000000"
+                        "expendable-pods-priority-cutoff": "-2147483648"
```

Comment:
Should be -2147483647? So we reserved -2147483648 as a priority which CA can ignore?